src/App.tsx

```diff
@@ -14,11 +14,93 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+type Props = {};
+type State = {
+  goodSelect: string[];
+};
+
+class App extends React.Component<Props, State> {
```

Comment:
Пропс можно сразу задать как пустой обьект:

```suggestion
type State = {
  goodSelect: string[];
};

class App extends React.Component<{}, State> {
```

Reply:
I agree with @kkiirill . What if you add some props tomorrow? You will create type Props with this prop, but now you just should add this prop to the exit type.